tests.py

```diff
@@ -686,6 +689,33 @@ def test_message_from_config_file(tmpdir, capsys, vcs):
 
     assert b'from-400.0.0-to-401.0.0' in tag_out
 
+@pytest.mark.parametrize(("vcs"), [xfail_if_no_git("git"), xfail_if_no_hg("hg")])
+def test_tag_message(tmpdir, vcs):
+    tmpdir.chdir()
+    check_call([vcs, "init"])
+    tmpdir.join("VERSION").write("42.3.1")
+    check_call([vcs, "add", "VERSION"])
+    check_call([vcs, "commit", "-m", "initial commit"])
+
+    main(['patch', '--current-version', '42.3.1', '--commit', '--tag', 'VERSION', '--tag-name', 'ReleasedVersion-{new_version}'])
+
+    tag_out = check_output([vcs, {"git": "tag", "hg": "tags"}[vcs]])
+    assert b'ReleasedVersion-42.3.2' in tag_out
+
+    if vcs == "git":  # TODO: hg
```

Comment:
I've revisited this and it appears there is no real ```git describe``` in mercurial so I don't think it's worth emulating.